ansible/devspace-runtime.yml

```diff
@@ -0,0 +1,36 @@
+- name: Run devspace
+  hosts: devspace
+  connection: ssh
+  gather_facts: no
+  become_user: "{{ devuser }}"
+
+  environment:
+    USER_ID: "{{ user_id }}"
+
+  tasks:
+    - name: Devspace down
+      docker_service:
+        project_src: "{{ devhome }}"
+        state: absent
```

Comment:
restart, that doesnt hurt if nothing is running

it will just say

```
TASK [Devspace down] ***********************************************************
Wednesday 21 September 2016  13:49:51 +0100 (0:00:00.028)       0:00:00.028 *** 
ok: [10.0.51.143]
```